multimedia/M2VDownsizer/Portfile

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                M2VDownsizer
+version             1.1
+revision            0
+
+categories          multimedia
+platforms           darwin
+license             GPL-2
+maintainers         {hotmail.com:amtor @RobK88} \
+                    openmaintainer
+
+description         MPEG-2 video stream recompressesion
+
+long_description    ${name} is a tool that re-compresses MPEG-2 (or m2v) video \
+                    streams.  In other words, ${name} will perform a complete \
+                    decompression and recompression of MPEG-2 video streams for the \
+                    best possible video quality.\
+                    \n \
+                    \n${name}  accepts the raw MPEG-2 (or m2v) video data (not VOB) \
+                    from the standard input and writes the recompressed video frames \
+                    to the standard output.\
+                    \n \
+                    \nBe  forewarned!  Decompression  and  recompression  could take \
+                    hours for large MPEG-2 video streams!  It is recommended that one \
+                    first tries transcoding MPEG-2 streams using M2VRequantiser since \
+                    transcoding is a lot faster and produces excellent results for most \
+                    MPEG-2 video streams. ${name} is only intended to be used for \
+                    large MPEG-2 video streams when transcoding does not produce \
+                    acceptable results.
+
+homepage            https://web.archive.org/web/20120508050929if_/http://www.metakine.com/
+master_sites        ${homepage}/files
```

Comment:
```suggestion
master_sites        ${homepage}files
```

Reply:
I do not know if this change will work.  I have to see if macports can find the source  after this change.  It was stored under the files sub-directory of the website.

Reply:
Ryan I do not understand this proposed change.  The download location is:
https://web.archive.org/web/20120512092147if_/http://www.metakine.com/files/M2VDownsizer.tgz

How could the port download the source from the above download location of the `/` is removed from ${master_sites}?

Reply:
On the preceding line you defined `homepage` to end with a slash:
```
homepage            https://web.archive.org/web/20120508050929if_/http://www.metakine.com/
```
Your proposed `master_sites` were:
```
master_sites        ${homepage}/files
```
After variable expansion that means `master_sites` will be set to `https://web.archive.org/web/20120508050929if_/http://www.metakine.com//files` which works but has an unnecessarily doubled slash before "files". My suggestion eliminates the unnecessary extra slash.